src/shared/tpm2-util.c

```diff
@@ -1471,7 +1672,8 @@ int tpm2_seal(const char *device,
                 return r;
 
         _cleanup_tpm2_handle_ Tpm2Handle *primary = NULL;
-        r = tpm2_make_primary(c, &primary, 0, &primary_alg);
+
+        r = tpm2_make_primary(c, 0, !!ret_srk_buf, &primary_alg, &primary);
```

Comment:
for new code we prefer if function parameters that are just literal values (i.e. "0"), get prefixed with the argument name to make it easier to read what this is:

```c
r = tpm2_make_primary(
                c,
                /* alg= / 0,
                !!ret_srk_buf,
                &primary_alg,
                &primary);
```

Reply:
ok so if I touch a hunk that's not following the current standard, update it.

Reply:
yeah, basically